src/sidebar/store/modules/test/activity-test.js

```diff
@@ -54,6 +54,19 @@ describe('sidebar/store/modules/activity', () => {
     });
   });
 
+  describe('isSavingAnnotation', () => {
+    beforeEach(() => {
+      store.annotationSaveStarted({ $tag: '1' });
+      store.annotationSaveStarted({ $tag: '2' });
+    });
+
+    it('returns `true` if annotation $tag is in current save requests', () => {
+      assert.isTrue(store.isSavingAnnotation({ $tag: '1' }));
+    });
+    it('returns `false` if annotation does not have a $tag');
```

Comment:
I think these 2 tests would be good to fill in

Reply:
Yeah, this is just plain whoops.